examples/Remarketing/UploadOfflineConversion.cs

```diff
@@ -0,0 +1,128 @@
+
+            // The conversion action ID.
+            long conversionActionId = long.Parse("INSERT_CONVERSION_ACTION_ID_HERE");
+
+            // The Click ID.
+            string gclid = "INSERT_GCLID_HERE";
+
+            // The conversion time.
+            string conversionTime = "INSERT_CONVERSION_TIME_HERE";
+
+            // The conversion value.
+            double conversionValue = double.Parse("CONVERSION_VALUE");
+
+            codeExample.Run(new GoogleAdsClient(), customerId, conversionActionId, gclid,
+                conversionTime, conversionValue);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example imports offline conversion values for specific " +
+                    "clicks to your account. To get Google Click ID for a click, use the " +
+                    "'click_view' resource: " +
+                    "https://developers.google.com/google-ads/api/fields/latest/click_view. " +
+                    "To set up a conversion action, run the AddConversionAction.cs example.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The Google Ads customer ID for the conversion action is
+        /// added.</param>
+        /// <param name="conversionActionId">The conversion action ID.</param>
+        /// <param name="conversionTime">The conversion time.</param>
+        /// <param name="gclid">The click ID.</param>
+        /// <param name="conversionValue">The convsersion value.</param>
+        public void Run(GoogleAdsClient client, long customerId, long conversionActionId,
+            string gclid, string conversionTime, double conversionValue)
+        {
+            // Get the ConversionActionService.
+            ConversionUploadServiceClient conversionUploadService =
+                client.GetService(Services.V2.ConversionUploadService);
+
+            // Creates a click conversion by specifying currency as USD.
+            ClickConversion clickConversion = new ClickConversion()
+            {
+                ConversionAction = ResourceNames.ConversionAction(customerId, conversionActionId),
+                Gclid = gclid,
+                ConversionValue = conversionValue,
+                ConversionDateTime = conversionTime,
+                CurrencyCode = "USD"
+            };
+
+            try
+            {
+                // Issues a request to upload the click conversion.
+                UploadClickConversionsResponse response =
+                    conversionUploadService.UploadClickConversions(
+                    customerId.ToString(), new[] { clickConversion }, true, false);
+
+                // Prints the result;
```

Comment:
I think this semicolon wants to be a period in the comment.

Reply:
Done.